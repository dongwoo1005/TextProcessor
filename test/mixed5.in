sample2.txt allcaps doublewords dropfirst 1
