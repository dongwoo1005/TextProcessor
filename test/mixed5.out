ELLO
ELLO
ORLD
ORLD
ELLO
ELLO
ORLD
ORLD
